### PR TITLE
Good night changed to Good evening on the login screen. (en)

### PR DESCRIPTION
**Pull request by muzzwood**
### What does it do?
Changes the English lexicon file for the manager login screen. Good night has been replaced with Good evening.

### Why is it needed?
 Good night is used as a farewell rather than a greeting in contemporary English.

### Related issue(s)/PR(s)
#14470

### CODE REVIEW

**Comment by muzzwood** [comment]
Hmm I switched branch for this new PR but it seems to be including the commit from my previous PR. 
What's the best course of action to prevent that?

Edit:
A bit of research seems to suggest I should've used `git stash` before switching to the new branch. Is that correct?

File: _build/templates/default/sass/_dashboard.scss [hunk 1]
```diff
@@ -87,6 +87,9 @@
       list-style: circle outside;
       padding: 0 0 0 15px;
     }
+    img {
+      max-width:100%;
```
Comment (JoshuaLuckers) [inline _build/templates/default/sass/_dashboard.scss]: I don't think this should be part of this PR.

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]

**Comment by JoshuaLuckers** [comment]
> Hmm I switched branch for this new PR but it seems to be including the commit from my previous PR.
> What's the best course of action to prevent that?
> 
> Edit:
> A bit of research seems to suggest I should've used `git stash` before switching to the new branch. Is that correct?

Always create a new branch (branch from 3.x in this case) for PR's.

**Comment by JoshuaLuckers** [comment]
If you pull in the latest changes from 3.x you probably don't have to change anything 😄

**Comment by Jako** [comment]
The following should work:

1. Fetch all changes from all remotes
2. Stash your changes
3. Checkout a new branch on base of 3.x
4. Rebase that branch with upstream/3.x
5. Pop your changes from the stash

```
git fetch --all
git stash
git checkout -b myFeature 3.x 
git rebase upstream/3.x
git stash pop
```

(This depends on that you have set up an upstream remote to modxcms/revolution)

After this, you could commit your stashed changes and create a PR.

**Comment by muzzwood** [comment]
Thanks all, I'm going to re-do this one.

**Pull request closed without merging**